client/grpc/grpc.go

```diff
@@ -4,6 +4,7 @@ package grpc
 
 import (
 	"context"
+	"github.com/uber/jaeger-client-go"
```

Comment:
It seems that the imports are not following the standard layout. check with go fmt or goimports which can be setup in any IDE.

Reply:
thank you, @mantzas, installed goimports and ran on the changed files